### PR TITLE
Release v0.22.0

### PR DESCRIPTION
**Pull request by nghialv**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: docs/content/en/blog/releases/v0.22.0.md [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+
+## Changelog since v0.21.1
+
+### New Features
+
+* Introduce [`trigger`](https://pipecd.dev/docs/user-guide/configuration-reference/#deploymenttrigger) field to give more controls on triggering the deployment
+    * `trigger.onCommit`: controls triggering new deployment when newly added Git commits touched the application
+    * `trigger.onCommand`: controls triggering new deployment when received a new `SYNC` command from the web console or `pipectl`
+    * `trigger.onOutOfSync`: controls triggering new deployment when application is at `OUT_OF_SYNC` state. Enabling this will force Piped to always attempt to keep the application as synced as possible.
+configuration-reference/#oncommit) instead.
```
Comment (khanhtc1202) [inline docs/content/en/blog/releases/v0.22.0.md:17]: Do we miss something here for the link?

Comment (nghialv) [reply docs/content/en/blog/releases/v0.22.0.md]: Nice catch! Removed it.

**Comment by khanhtc1202** [comment]
Go go go
/lgtm

**Comment by nakabonne** [comment]
And go
/approve

**Pull request merged**